### DISCUSSION TITLE
New 16.2 version doesn't comply with GraphQl spec for null fields

### DISCUSSION THREAD

**Discussion by miman** [graphql-java/graphql-java#2212] [Q&A] [answered]
This question is obsolete, the issue we assumed was due to this lib was due to a Jackson not_null configuration done in our internal framework.

----------------------------
In 16.1 any null field was present in the response
ex: 
{
   name: null,
   age: 10
}

but in 16.2 the entire null field is omitted:
{
   age: 10
}
This doesn't work with the Apollo client (an according to the Apollo team doesn't comply with the GraphQl spec which indicates that any null field should be present in the response).

Is there any way to configure the system to generate the null fields ?

**Comment by andimarek** [comment, marked as answer]
Can you provide more informations about the query, schema and the result of the Datafetchers?

I don't think the other issue is related: it is about schema Printing.

**Reply by miman** [reply]
Ignore this question, this error was due to an change in Jackson configuration that had been done in our inhouse framework and was not due to this library as I first suspected.
